scripts/validate_job_status_ps.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
Why not modifying existing script instead of copying + modifying?

Reply:
Umm, we have master+ps+worker case, and master+worker case. Now, the validate job status script is hard-code. Let me see how to make it more general.

Reply:
I will fix this issue in next PR, which provides a general validate job status script.